apps/test/unit/util/imageUtilsTest.js

```diff
@@ -26,4 +31,86 @@ describe('image utils', () => {
       });
     });
   });
+
+  describe('toCanvas', () => {
+    it('returns a Canvas unchanged', async () => {
+      const canvas = document.createElement('canvas');
+      const result = await toCanvas(canvas);
+      assert(canvas === result);
+    });
+
+    it('converts an image element to a canvas', async () => {
+      const image = await imageFromURI(expectedChromePng);
+      const result = await toCanvas(image);
+      assert.instanceOf(result, HTMLCanvasElement);
+      assertVisualMatch(image, result);
+    });
+
+    it('converts an image URI to a canvas', async () => {
+      assert.typeOf(expectedChromePng, 'string');
+      const result = await toCanvas(expectedChromePng);
+      assert.instanceOf(result, HTMLCanvasElement);
+      assertVisualMatch(expectedChromePng, result);
+    });
+
+    it('converts a data URI to a canvas', async () => {
+      const result = await toCanvas(TEST_DATA_URI);
+      assert.instanceOf(result, HTMLCanvasElement);
+      assertVisualMatch(TEST_DATA_URI, result);
+    });
+  });
+
+  describe('toImageData', () => {
+    it('returns ImageData unchanged', async () => {
+      const imageData = new ImageData(8, 8);
```

Comment:
Running in Phantom on Drone:
```
ImageDataConstructor is not a constructor (evaluating 'new ImageData(8, 8)')
```
I'll have to find another way to test this in both browsers.